docs/data/pages.ts

```diff
@@ -28,88 +28,115 @@ const pages: MuiPage[] = [
   {
     pathname: '/x/react-data-grid-group',
     title: 'Data Grid',
-
     children: [
       { pathname: '/x/react-data-grid', title: 'Overview' },
-      { pathname: '/x/react-data-grid/demo' },
       { pathname: '/x/react-data-grid/getting-started' },
-      { pathname: '/x/react-data-grid/layout' },
+      { pathname: '/x/react-data-grid/demo' },
+      { pathname: '/x/react-data-grid/faq', title: 'FAQs' },
       {
-        pathname: '/x/react-data-grid/columns',
+        pathname: '/x/react-data-grid/core-features-group',
+        subheader: 'Core features',
```

Comment:
Nitpick, until a few years ago, people used `@material-ui/core` to install the Material UI components.
Is there a risk that "core" still means "from the `@mui/material` package" (which is the current name of `@material-ui/core`).
That would be a terrible confusion :laughing:

Reply:
Yeah you're right, I was a little concerned about that as well so I'll come up with some other synonyms for this. The main motivation is to standardize the "(Common) features" section across all the X products, but the name "Common features" only makes sense for the multi-component packages. Other options:

- Primary features
- Key features
- Main features
- Basic features

Reply:
I like "Main features" as its very easily to understand what it means.
With synonyms we also have

- Fundamental features
- Essential features

Reply:
Sounds good, I'm happy with "Main features" so I'll run with that here.